pkg/resources/job.go

```diff
@@ -284,7 +293,7 @@ func resourceJobUpdate(ctx context.Context, d *schema.ResourceData, m interface{
 		d.HasChange("target_name") || d.HasChange("execute_steps") || d.HasChange("run_generate_sources") ||
 		d.HasChange("generate_docs") || d.HasChange("triggers") || d.HasChange("schedule_type") ||
 		d.HasChange("schedule_interval") || d.HasChange("schedule_hours") || d.HasChange("schedule_days") ||
-		d.HasChange("schedule_cron") || d.HasChange("deferringJobId") {
+		d.HasChange("schedule_cron") || d.HasChange("deferringJobId") || d.HasChange("selfDeferring") {
```

Comment:
should be the self_deferring I think? (the deferringJobId prior is wrong 😬 )

Reply:
Fixed both!